pkg/tsdb/testdatasource/testdata/flight-simple-X.txt

```diff
@@ -1,7 +1,7 @@
 🌟 This was machine generated.  Do not edit. 🌟
 
-Frame[0] 
-Name: 
+Frame[0]
```

Comment:
Why did this change?

Reply:
That's related to a PR merged by Ryan, was needed back when I started my PR, will not be needed when I merge with main anymore :)